### DISCUSSION TITLE
Syncing

### DISCUSSION THREAD

**Discussion by Shahin-rmz** [LinwoodDev/Butterfly#87] [Ideas]
First of all, thanks for the awesome app and design specially being multy platform. 
I use the app on my android, and(want to) use the app on my (future) touchscreen linux tablet.
It would be awesome if I can synch the app  between my devices without clicking export and go to directory every time.
Some automated, or one touch syncing with clouds (for example next cloud) would be awesome.

**Comment by CodeDoctorDE** [comment]
Good idea

**Comment by CodeDoctorDE** [comment]
I created an issue (#88 ). It's a big enhancement, I will look into it after I fixed the performance issues.
